frontend/src/EventDetails/EventDetails.module.css

```diff
@@ -0,0 +1,102 @@
+.container {
+  background: hsl(0, 0%, 100%);
+  border-radius: 16px;
+  padding: 0;
```

Comment:
⚠️ action required: does the element `.container` is a class of have padding by default? is this style needed?